docs/howto_2_CustomizedTuner.md

```diff
@@ -6,7 +6,7 @@ So, if user want to implement a customized Tuner, she/he only need to:
 
 1) Inherit a tuner of a base Tuner class
 2) Implement receive_trial_result and generate_parameter function
-3) Write a script to run Tuner
+3) Configure your customized tuner in experiment yaml config file
```

Comment:
why this file name("howto_2_CustomizedTuner") is so strange....

Reply:
renamed by @scarlett2018